packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -168,6 +168,14 @@ const SavedChartsHeader: FC = () => {
         health.data?.auth.google.oauth2ClientId !== undefined &&
         health.data?.auth.google.googleDriveApiKey !== undefined;
 
+    const cannotAccessGoogleSheets = user.data?.ability?.cannot(
```

Comment:
Could you use the `<Can>` component for permissions since it will pick up Account abilities.